core/src/main/java/com/bankbazaar/core/model/TripActivityCompositeKey.java

```diff
@@ -0,0 +1,16 @@
+package com.bankbazaar.core.model;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
```

Comment:
@Data itself handles the constructors

Reply:
I tried removing @AllArgsConstructor and @NoArgsConstructor but im getting no contructor error